src/lib/emojiKeywords.js

```diff
@@ -175,5 +175,6 @@ export default {
   "cold": "snowflake",
   "tailwind": "tailwind",
   "tailwindcss": "tailwind",
-  "c": "c"
+  "c": "c",
+  "nft": "squaresupply"
```

Comment:
Personally, I'd prefer to use general emojis for general topics or a least NFTs with a generalised name so we get `/r/nft` etc.

Reply:
squaresupply is an NFT project by zfogg.

Reply:
i actually agree with @sampoder here, @tanishq-soni. make "square supply", "squaresupply.xyz", and "squaresupply" work for the `:squaresupply:` emoji.. the word "NFT" is too broad to apply to the `:squaresupply:` emoji lol

Reply:
okay 👍🏻